vertx-mongo-client/src/main/kotlin/io/vertx/kotlin/ext/mongo/MongoClientChange.kt

```diff
@@ -0,0 +1,9 @@
+package io.vertx.kotlin.ext.mongo
```

Comment:
I assume this is generated code?  Same goes for the other Kotlin pieces

Reply:
Yes